modules/builtin/pulseaudio.go

```diff
@@ -0,0 +1,110 @@
+package builtin
+
+import (
+	"log"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+	"github.com/mattboll/pulseaudio"
+
+	"fyne.io/fynedesk"
+	wmtheme "fyne.io/fynedesk/theme"
+)
+
+var pulseaudioMeta = fynedesk.ModuleMetadata{
+	Name:        "Pulseaudio",
```

Comment:
This string (and the overall naming) I think would be better as "Audio" or "Sound" - it is specific to pulse just now, but over time this module would surely manage all audio for fynedesk?

Reply:
Pulseaudio plugin and alsa plugin could be slightly different, I'm not sure yet but it's possible that alsa plugin can't change sound by application, ok right now it's not possible with the pulseaudio widget too but I hope that will change
I also looked at xfce which has both plugins and not one sound plugin
That's why I named Pulseaudio but I can change my mind